ironmq/src/main/scala/akka/stream/alpakka/ironmq/Codec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ironmq
+
+import io.circe.{ Decoder, Encoder, Json }
+import io.circe.syntax._
```

Comment:
Hm... I guess it's ok, we won't be able to stick to a single JSON lib I guess - and circe is nice (though we usually used spray)